sdk/js/src/utils/consts.ts

```diff
@@ -815,28 +809,28 @@ export function isEVMChain(
   chain: ChainId | ChainName
 ): chain is EVMChainId | EVMChainName {
   const chainName = coalesceChainName(chain);
-  return EVMChainNames.includes(chainName);
+  return EVMChainNames.includes(chainName as unknown as EVMChainName);
```

Comment:
Is it necessary to cast to `unknown` here (and below)? Seems to build for me when we don't do this

Reply:
It was to make my editor happy, otherwise it cries about `EVMChainNames` having the subset and `chainName` being the full set. 

When it was ReadOnlyArray it was fine because it was (accidentally) evaluating to the full set of chain names despite the values being a subset.